docker-compose.ci.yml

```diff
@@ -0,0 +1,9 @@
+tests:
+  build: .
```

Comment:
Cool - I totally, I wouldn't have noticed except for john's message yesterday.   I'll have to have another look at the plain-pipeline project (as it does lots of cool automated things behind the scenes - that are constantly changing it seems :)